project/src/main/java/com/google/sps/data/CompetitionSummary.java

```diff
@@ -0,0 +1,38 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+import com.google.auto.value.AutoValue;
+
+/**
+ * A summary of a competition the user is in
+ */
+@AutoValue
+public abstract class CompetitionSummary {
+  public static CompetitionSummary create(long id, String competitionName, String organiserName, String organiserEmail, 
+  long startDate, long endDate, int rank, int rankYesterday, int netWorth, int amountAvailable) {
```

Comment:
Wait I thought rankYesterday wasn't possible - is this still a thing?

Reply:
Or has the rework made this possible

Reply:
Yeah I can do it now after the adjustments I made